main/provider/index.ts

```diff
@@ -533,7 +533,7 @@ export class Provider extends EventEmitter {
           account: (currentAccount as FrameAccount).id,
           origin: payload._origin,
           approvals: [],
-          feesUpdatedByUser: feesUpdated || false,
+          feesUpdatedByUser: false,
```

Comment:
@Jamchello can you explain this change? seems like we're changing the functionality here